build/app.js

```diff
@@ -5200,7 +5200,7 @@ var module = module || undefined;
 
 var __provisioner = {
   provision_contracts: function(scope) {
-    scope.__contracts = JSON.parse("{\n  \"Example\": {\n    \"source\": \"/Users/tim/Documents/workspace/Consensys/hooked-web3-provider/contracts/Example.sol\",\n    \"binary\": \"0x60606040525b5b60888060136000396000f30060606040526000357c010000000000000000000000000000000000000000000000000000000090048063130d6dc514604157806360fe47b114606057603f565b005b604a6004506071565b6040518082815260200191505060405180910390f35b606f600480359060200150607a565b005b60006000505481565b806000600050819055505b5056\",\n    \"abi\": [\n      {\n        \"inputs\": [],\n        \"constant\": true,\n        \"name\": \"test_value\",\n        \"outputs\": [\n          {\n            \"type\": \"uint256\",\n            \"name\": \"\"\n          }\n        ],\n        \"type\": \"function\"\n      },\n      {\n        \"inputs\": [\n          {\n            \"type\": \"uint256\",\n            \"name\": \"new_value\"\n          }\n        ],\n        \"constant\": false,\n        \"name\": \"set\",\n        \"outputs\": [],\n        \"type\": \"function\"\n      },\n      {\n        \"inputs\": [],\n        \"type\": \"constructor\"\n      }\n    ]\n  }\n}");
+    scope.__contracts = JSON.parse("{\n  \"Example\": {\n    \"source\": \"/Users/ck/consensys/noncense/persona-integration/myhooked/contracts/Example.sol\"\n  }\n}");
```

Comment:
Hey @ckeenan. I don't think this change should have made it in. Can you tell me what version of Truffle you're using? Regardless, the build should be run using the latest version, as both the before and after have issues with the contract's source path.